website/docs/help-and-support/troubleshooting-guide/deployment-errors.mdx

```diff
@@ -135,6 +135,55 @@ As the error message in the server logs suggests, your Appsmith instance has mis
 
 #### Solution
 
-To resolve this error, you must [upgrade to the checkpoint version](/getting-started/setup/instance-management#checkpoint-upgrade), in this case, `v1.9.2`.
+To resolve this error, you must [upgrade to the checkpoint version](/getting-started/setup/instance-management/update-appsmith#upgrade-to-a-checkpoint-version), in this case, `v1.9.2`.
 
-To prevent similar issues in the future, consider [enabling auto updates](/getting-started/setup/installation-guides/docker#enabling-appsmith-auto-updates) for your Appsmith instance. If you already had auto updates on and still encountered this issue, please contact the [Appsmith team](https://discord.com/invite/rBTTVJp) to determine the cause of the deployment problem
\ No newline at end of file
+To prevent similar issues in the future, consider [enabling auto updates](/getting-started/setup/installation-guides/docker#enabling-appsmith-auto-updates) for your Appsmith instance. If you already had auto updates on and still encountered this issue, please contact the [Appsmith team](https://discord.com/invite/rBTTVJp) to determine the cause of the deployment problem
+
+## Illegal instruction (core dumped)
+MongoDB fails to start, after migrating to MongoDB version 5.
+
+#### Error message
+
+<Message
+messageContainerClassName="error" 
+messageContent="Illegal instruction (core dumped) mongod –fork –port 27017 –dbpath “$MONGO_DB_PATH"></Message>
+
+#### Cause
+MongoDB version 5 has specific CPU requirements. If these requirements are not met the server wouldn't start.
+
+#### Solution
+To resolve this error, you must upgrade your infrastructure. See the [List of complaint hardware](https://www.mongodb.com/docs/manual/administration/production-notes/#platform-support).
```

Comment:
Currently, we have 2 options, either upgrade hardware to support Mongov5 or rollback to old Appsmith v.18.
Lets add a third option, that is to provision and use an external Mongov5 (preferably on Mongo Atlas)

Reply:
@jnikhila can we add this?